sparkmagic/sparkmagic/utils/configuration.py

```diff
@@ -237,6 +237,20 @@ def configurable_retry_policy_max_retries():
     return 8
 
 
+@_with_override
+def spark_statement_errors_are_fatal():
+    # If set to true, any spark statement errors will be considered
+    # fatal and will raise a SparkStatementException
+    return False
+
+
+@_with_override
+def shutdown_session_on_spark_statement_errors():
```

Comment:
Is there some proper way for setting this (or any sparkmagic configuration) other than the `~/.sparkmagic/config.json` file? 

I couldn't find anything, so went with this hack for now:

```python
from sparkmagic.utils import configuration

def spark_statement_errors_are_fatal():
    return True
# replace the method in imported class:
setattr(configuration, 'spark_statement_errors_are_fatal', spark_statement_errors_are_fatal)
```

Reply:
I'll open an issue for this.